src/components/main/BannerCarousel.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from "react";
+import { Carousel, Chip } from "jci-moyeo-design-system";
+import Image from "next/image";
+import styled from "styled-components";
+
+const BannerCarousel = () => {
+  const [mount, setMount] = useState(false);
+
+  useEffect(() => {
+    setMount(true);
+  }, []);
+
+  if (!mount) return null;
+  return (
+    <Container>
+      <Carousel className="carousel">
+        <ItemContainer>
+          <ContentWrapper>
+            <ChipWrapper>
+              <Chip
+                variants="pill"
+                color="active"
+                label="고생 끝에 낙이 온다"
+                outlined
+              />
+              <Chip
+                variants="pill"
+                color="active"
+                label="모여로 다 모여"
+                outlined
+              />
+            </ChipWrapper>
+            <TitleWrapper>
+              <Title>사이드 프로젝트/스터디 찾고 있어?</Title>
+              <Title>
+                지금 모여로 다 <span>모여</span>!
+              </Title>
+            </TitleWrapper>
+          </ContentWrapper>
+          <ImageWrapper>
+            <Image
+              src={"/assets/banner/banner.webp"}
+              layout="fill"
+              priority
+              alt={"banner"}
+            />
+          </ImageWrapper>
+        </ItemContainer>
+      </Carousel>
+    </Container>
+  );
+};
+
+export default BannerCarousel;
+
+const Container = styled.section`
+  height: 252px;
+
+  background: ${({ theme }) => theme.colors.primary["100"]};
+
+  @media (max-width: ${({ theme }) => `${theme.breakpoints.md}px`}) {
+    height: 152px;
+
+    .carousel {
+      height: 152px;
+    }
+  }
+
+  .carousel {
+    max-width: 1220px;
+    margin: auto;
+  }
+`;
+
+const ItemContainer = styled.div`
+  display: flex !important;
+  position: relative;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+
+  @media (max-width: ${({ theme }) => `${theme.breakpoints.md}px`}) {
+    padding: 0 16px;
+  }
+`;
```

Comment:
```suggestion
const ItemContainer = styled.div`
  height: 252px;
  display: flex !important;
  position: relative;
  width: 100%;
  align-items: center;
  justify-content: space-between;

  @media (max-width: ${({ theme }) => `${theme.breakpoints.md}px`}) {
    height: 152px;
    padding: 0 16px;
  }
`;
```
ItemContainer의 높이를 반응형 처리해야 할 것 같아요. 모바일에서 깨지네요ㅠㅠ
반응형 처리를 했는데 indicator는 밖으로 삐져나요네요ㅠㅠ
<img width="493" alt="스크린샷 2022-10-21 오전 12 29 59" src="https://user-images.githubusercontent.com/12439567/196992726-72e1dc85-9ab5-4f0d-a17d-8bb5c559940b.png">

Reply:
오잉,,반영하였습니다! 감사합니당